shared_model/cryptography/blob.hpp

```diff
@@ -20,30 +20,65 @@
 
 #include "interfaces/model_primitive.hpp"
 #include "utils/string_builder.hpp"
+#include "utils/lazy_initializer.hpp"
 
 namespace shared_model {
   namespace crypto {
 
+    static const std::string code = {'0',
```

Comment:
why you didn't use just string literal: `"0123.."`

Reply:
agree, it was copied from old blob impementation